samples/bot-sso-adaptivecard/nodejs/index.js

```diff
@@ -0,0 +1,59 @@
+const path = require('path');
+const express = require('express');
+const ENV_FILE = path.join(__dirname, '.env');
+require('dotenv').config({ path: ENV_FILE });
+const PORT = process.env.PORT || 3978;
+const server = express();
+server.use(express.json());
+server.use(express.urlencoded({
+    extended: true
+}));
+
+// Import required bot services.
+// See https://aka.ms/bot-services to learn more about the different parts of a bot.
+const { BotFrameworkAdapter } = require('botbuilder');
+
+const { BotSSOAdativeCard } = require('./bots/botSSOAdaptiveCard');
+
+// Create adapter.
+// See https://aka.ms/about-bot-adapter to learn more about adapters.
+const adapter = new BotFrameworkAdapter({
+    appId: process.env.MicrosoftAppId,
+    appPassword: process.env.MicrosoftAppPassword,
+    connectionName :process.env.ConnectionName
+});
+
+adapter.onTurnError = async (context, error) => {
+    // This check writes out errors to console log .vs. app insights.
+    // NOTE: In production environment, you should consider logging this to Azure
+    //       application insights. See https://aka.ms/bottelemetry for telemetry 
+    //       configuration instructions.
+    console.error(`\n [onTurnError] unhandled error: ${error}`);
+
```

Comment:
Need to add below lines
  // Uncomment below commented line for local debugging.
    // await context.sendActivity(`Sorry, it looks like something went wrong. Exception Caught: ${error}`);

Reply:
fixed